src/games/kalambury/Board/GameBoard.js

```diff
@@ -27,6 +29,29 @@ const GameBoard = ({ guess, setGuess, envokeLastAnswer, guessInputRef }) => {
     success: false,
   };
 
+  const throttledSendAppend = useMemo(
+    () =>
+      throttleAccumulate(sendChatMessage, 200, ([acc], [args]) => [
```

Comment:
This gives only 5fps which is noticeably not smooth. Since we're now only sending small chunks of data we can try to send more frequent updates at the beginning. Locally I saw no slowdowns when sending 30 updates per second and it appeared really smooth. We could always reduce this if there are issues in the future

Reply:
ok 👍 I'll change that to 50ms. This should be ok over network